sources/platform/storage/request_queue.md

```diff
@@ -274,6 +274,195 @@ Example payload:
 
 For further details and a breakdown of each storage API endpoint, refer to the [API documentation](/api/v2#/reference/key-value-stores).
 
+## Features {#features}
+
+Request queue is a storage type built with scraping in mind, enabling developers to write scraping logic efficiently and scalable.
+The Apify tooling, including [Crawlee](https://crawlee.dev/) and [Apify SDKs](https://docs.apify.com/sdk/js/), incorporates all these features, enabling users to leverage them effortlessly without extra configuration.
```

Comment:
Is this feature available in Python SDK as well? If so I would add separate link something like:
```suggestion
The Apify tooling, including [Crawlee](https://crawlee.dev/), [Apify JS SDK](https://docs.apify.com/sdk/js/), and [Apify Python SDK](https://docs.apify.com/sdk/python/) , incorporates all these features, enabling users to leverage them effortlessly without extra configuration.
```

Once we have a unified landing page for SDKs we can change it for a singular link.